cmake/ElektraCache.cmake

```diff
@@ -365,14 +365,8 @@ set (LIB_SUFFIX ""
 		"Optional suffix to use on lib folders (e.g. 64 for lib64)"
     )
 
-set (MEMORYCHECK_COMMAND
```

Comment:
strange, actually your `-DMEMORYCHECK_COMMAND` should take preference?
Or did you try the cmake GUI? Nevertheless, simply put an `if` around it would be less invasive, but I think you can can override it even without.